eZ/Publish/SPI/Search/FullTextFieldValue.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/**
+ * This file is part of the eZ Publish Kernel package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\SPI\Search;
+
+/**
+ * The wrapper for \eZ\Publish\SPI\Search\Field of the type \eZ\Publish\SPI\Search\FieldType\FullTextField.
+ */
+class FullTextFieldValue extends Field
+{
+    /**
+     * @param mixed $value
+     */
+    public function __construct($value)
+    {
+        parent::__construct('fulltext', (string) $value, new FieldType\FullTextField());
```

Comment:
You mean in `getFulltextData()`? I thought you went with returning an array of strings there?
This basically hardcodes field name, and we might need it for other stuff, like indexing other full text fields (per Content field, then implementing `FULLTEXT` operator for `Field`/`CustomField` criterion).

Also, since this is API (SPI), I would do this kind of thing in the implementation and not here.